### PR TITLE
simplify plugin unzipping log info to one line 

### PR DESCRIPTION
**Pull request by kpge**
#### What it does

fix: #6138

#### How to test

Simplify plugin unzipping log info to one line :
image
* create plugins folder under Theia repo folder
* Google for an extension (*.vsix file) and download it to the plugins folder
* start Theia browser example
* check the log in terminal or console

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

File: packages/plugin-ext-vscode/src/node/plugin-vscode-file-handler.ts [hunk 1]
```diff
@@ -37,10 +37,9 @@ export class PluginVsCodeFileHandler implements PluginDeployerFileHandler {
     }
 
     async handle(context: PluginDeployerFileHandlerContext): Promise<void> {
-        // need to unzip
-        console.log('unzipping the plugin', context.pluginEntry());
-
         const unpackedPath = path.resolve(this.unpackedFolder, path.basename(context.pluginEntry().path()));
+        console.log(`unzipping the plugin '${path.basename(context.pluginEntry().path())}' to directory: ${unpackedPath}`);
```
Comment (benoitf) [inline packages/plugin-ext-vscode/src/node/plugin-vscode-file-handler.ts]:
```suggestion
        console.log(`unzipping the VS Code extension '${path.basename(context.pluginEntry().path())}' to directory: ${unpackedPath}`);
```

File: packages/plugin-ext/src/main/node/handlers/plugin-theia-file-handler.ts [hunk 2]
```diff
@@ -32,10 +32,9 @@ export class PluginTheiaFileHandler implements PluginDeployerFileHandler {
     }
 
     async handle(context: PluginDeployerFileHandlerContext): Promise<void> {
-        // need to unzip
-        console.log('unzipping the plugin', context.pluginEntry());
-
         const unpackedPath = path.resolve(this.unpackedFolder, path.basename(context.pluginEntry().path()));
+        console.log(`unzipping the plugin '${path.basename(context.pluginEntry().path())}' to directory: ${unpackedPath}`);
```
Comment (benoitf) [inline packages/plugin-ext/src/main/node/handlers/plugin-theia-file-handler.ts]:
```suggestion
        console.log(`unzipping the VS Code extension '${path.basename(context.pluginEntry().path())}' to directory: ${unpackedPath}`);
```

Comment (akosyakov) [reply packages/plugin-ext/src/main/node/handlers/plugin-theia-file-handler.ts]: It is the Theia plugin handler.

**Comment by vince-fugnitto** [comment]
@akosyakov do you know how a plugin can fail resolving or fail to load? 
I'd like to test in that case if we get a meaningful message.

Also, I don't know if we agreed on what format and params should be present when logging the unzipping of plugins, but is it sufficient to display the path, and copy path only?

File: packages/plugin-ext/src/main/node/handlers/plugin-theia-file-handler.ts (hunk 2 above)
Comment (benoitf) [reply packages/plugin-ext/src/main/node/handlers/plugin-theia-file-handler.ts]: yes updated ! thx

**Comment by akosyakov** [comment]
@vince-fugnitto try to point to something what is no zip file, but has vsix extension

Having a source and target paths is enough to me, as it is done in the PR.

**Comment by lmcbout** [comment]
Question, I tried to install in VSCode the following plugin I get the following error: 
`Unable to install 'jabacchetta.vscode-essentials' extension because it is not compatible with the current version of VS Code (version 1.37.1).`

 but if I unzip it in Theia, I don't get an indication whether the plugin works or not ?
Should we have an indication if the plugin is usable or not ?

**Comment by akosyakov** [comment]
You should be able to see all deployed plugins in `Plugins` view. If it is not there it is an indicator. For any plugin which is there you will receive an error notification if it cannot be loaded or activated.

**Comment by benoitf** [comment]
@lmcbout AFAIK the new default for VS Code version is 1.38
you can override the check with VSCODE_API_VERSION env variable where you run theia.

https://github.com/theia-ide/theia/pull/6112

**Comment by kpge** [comment]
> I'd like to test in that case if we get a meaningful message.

I didn't change the error log in this PR, and it will show the failed log like this:
image

**Comment by akosyakov** [comment]
@kpge please squash suggestions commit

**Comment by kpge** [comment]
> @kpge please squash suggestions commit

@akosyakov ok, I have rebased it.

**Pull request merged**